### DISCUSSION TITLE
tieredImageNet

### DISCUSSION THREAD

**Discussion by 7xliu** [RL-VIG/LibFewShot#35] [Q&A] [answered]
用tieredImageNet数据集跑代码显示：/pytorch/aten/src/ATen/native/cuda/Loss.cu:247: nll_loss_forward_reduce_cuda_kernel_2d: block: [0,0,0], thread: [0,0,0] Assertion `t >= 0 && t < n_classes` failed.   RuntimeError: CUDA error: device-side assert triggered
这个错误，请问怎么解决

**Comment by wZuck** [comment, marked as answer]
你好，可以将你的配置文件贴上来吗，方便我们排查。
